.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -0,0 +1,194 @@
+name: Arm64-CI
+
+# github.repository has been used to ensure CI is only run on the repo where
+# self-hosted runners are installed. This will prevent Arm64 CI failing on forks
+
+on: [push, pull_request]
+env:
+  ARCH: arm64
+  CC: gcc
+  CONTAINER_NAMESPACE: ghcr.io/opendataplane/odp-docker-images
+  OS: ubuntu_20.04
+
+jobs:
+
+  Build_arm64_native:
+    if: ${{ github.repository == 'OpenDataPlane/odp' }}
+    runs-on: self-hosted
+    strategy:
+      fail-fast: false
+      matrix:
+        cc: [gcc, clang]
+        conf: ['', 'CFLAGS=-O3', 'CFLAGS=-O1', 'CFLAGS=-O0 --enable-debug=full', '--enable-lto', '--enable-lto --disable-abi-compat', '--enable-pcapng-support']
+        exclude:
+          - cc: clang
+            conf: '--enable-lto'
+          - cc: clang
+            conf: '--enable-lto --disable-abi-compat'
+    steps:
+      - uses: AutoModality/action-clean@v1.1.0
+      - uses: actions/checkout@v2
+      - run: sudo docker run -i -v `pwd`:/odp --privileged --shm-size 8g -e CC="${{matrix.cc}}"
+               -e CONF="${{matrix.conf}}" $CONTAINER_NAMESPACE/odp-ci-${OS}-${ARCH}-native /odp/scripts/ci/build_${ARCH}.sh
```

Comment:
`build_arm64.sh` ends up calling `build.sh`, which still assumes an x86 host. It always sets `--build=x86_64-linux-gnu` to ODP configure and the dynamically linked test application is not run if `TARGET_ARCH` is set.